<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plotter</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0.51705, y1:0, x2:0.468, y2:1, stop:0 rgba(1, 42, 74, 255), stop:1 rgba(255, 255, 255, 255));
}
#Title{
margin :10px;
Font:24px;
max-height: 55%;
}

#Title ,#groupBox_2 ,#groupBox , #label_2, #label_3, #label_4 , #Clear_Button , #PlotButton{
Color:white;
font-Weight:900;
}
#label_5 , #label_6 , #label_7{
    color: rgb(97, 165, 194);
	font-Weight:900;
}

#groupBox_2 ,#groupBox , #Clear_Button ,#PlotButton , #MinValue , #MaxValue, #Function{
  border: 1px solid rgb(97, 165, 194);
  border-radius: 10px;
}


#Clear_Button , #PlotButton{
	background-color: rgb(97, 165, 194);
	border: 1px solid rgb(97, 165, 194);
	border-radius: 5px;
	max-width: 100 rem;
}


#Function:hover, #MaxValue:hover, #MinValue:hover{
color: rgb(97, 165, 194);
background-color: rgb(97, 165, 194);
color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="Title">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>99</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Welcome To Plotter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>99</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>F(X) = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Function">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="PlotButton">
          <property name="font">
           <font>
            <weight>99</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Clear_Button">
          <property name="font">
           <font>
            <weight>99</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <weight>99</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Range</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>99</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Min(X) = </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MinValue">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <weight>99</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Max(X) =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxValue">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <weight>99</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>99</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Supported Operator (* , / , +, - , ^)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>99</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Use Letter x or X to experss your function </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <weight>99</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>e.g 3*x - 4 , x^3 - 2 * x</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
